modules/kargoBidAdapter.js

```diff
@@ -28,57 +28,77 @@ export const spec = {
   },
   buildRequests: function(validBidRequests, bidderRequest) {
     const currencyObj = config.getConfig('currency');
-    const currency = (currencyObj && currencyObj.adServerCurrency) || 'USD';
-    const bidIDs = {};
-    const bidSizes = {};
+    const currency = (currencyObj && currencyObj.adServerCurrency);
```

Comment:
I think this would evaluate to a boolean? Is this suppose to be a ternary?
```suggestion
    const currency = (currencyObj && currencyObj.adServerCurrency) ? currencyObj.adServerCurrency : null;
```